packages/twenty-server/src/engine/core-modules/workflow/services/workflow.service.ts

```diff
@@ -0,0 +1,111 @@
+import { InjectRepository } from '@nestjs/typeorm';
+import { Injectable } from '@nestjs/common';
+
+import { join } from 'path';
+
+import { Repository } from 'typeorm';
+
+import { ServerlessFunctionService } from 'src/engine/metadata-modules/serverless-function/serverless-function.service';
+import { CodeIntrospectionService } from 'src/modules/code-introspection/code-introspection.service';
+import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
+import {
+  WorkflowTrigger,
+  WorkflowTriggerType,
+} from 'src/modules/workflow/workflow-trigger/types/workflow-trigger.type';
+import {
+  WorkflowActionType,
+  WorkflowStep,
+} from 'src/modules/workflow/workflow-executor/types/workflow-action.type';
+import { isDefined } from 'src/utils/is-defined';
+import { generateFakeObjectRecordEvent } from 'src/engine/core-modules/event-emitter/utils/generate-fake-object-record-event';
+import { INDEX_FILE_NAME } from 'src/engine/core-modules/serverless/drivers/constants/index-file-name';
+
+@Injectable()
+export class WorkflowService {
```

Comment:
this should live in the module `workflow`, not in core-modules. We only keep in core-modules the resolver because current archi forces us to do so.

Maybe another sub-module, `WorkflowStepOutputSchema`?

Reply:
ok